kolibri/core/assets/src/styles/core-theme.scss

```diff
@@ -20,7 +20,7 @@ $core-text-disabled: #dfdfdf;
 $core-bg-warning: #fff3e1;
 
 $core-text-error: #b93329;
-$core-bg-error: #fee9e7;
+$core-bg-error: #eeeeee;
```

Comment:
We might want to just get rid of this as a global variable.

Reply:
agreed